Tests are failing under Django 1.11

#606 is adding Django 1.11 to the testing matrix, but with the result ignored since the tests are currently failing:
https://travis-ci.org/carltongibson/django-filter/builds/193589440#L294

eg
```
FAIL: test_get_filtered_detail_view (tests.rest_framework.test_backends.IntegrationTestDetailFiltering)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/carltongibson/django-filter/tests/rest_framework/test_backends.py", line 399, in test_get_filtered_detail_view
    self.assertEqual(response.data, data)
AssertionError: {u'id': 1, 'text': u'aaa', 'decimal': u'0.25', u'basefilterableitem_ptr': 1, 'date': '2012-10-08'} != {u'id': 1, u'text': u'aaa', u'decimal': '0.25', u'date': '2012-10-08'}
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,7 +1,7 @@
 sudo: false
 
 language: python
-python: "3.6"
+python: "3.5"
 
 cache: pip
 

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,12 +1,7 @@
 sudo: false
 
 language: python
-python:
-  - "2.7"
-  - "3.3"
-  - "3.4"
-  - "3.5"
-  - "3.6"
+python: "3.6"
 
 cache: pip
 

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,7 +1,13 @@
 sudo: false
 
 language: python
-python: '3.5'
+python:
+  - "2.7"
+  - "3.3"
+  - "3.4"
+  - "3.5"
+  - "3.6"
+
 cache: pip
 
 install:

--- a/README.rst
+++ b/README.rst
@@ -14,7 +14,7 @@ Requirements
 
 * **Python**: 2.7, 3.3, 3.4, 3.5
 * **Django**: 1.8, 1.9, 1.10
-* **DRF**: 3.4, 3.5
+* **DRF**: 3.5
 
 Installation
 ------------

--- a/docs/guide/install.txt
+++ b/docs/guide/install.txt
@@ -15,7 +15,7 @@ Requirements
 ------------
 
 Django-filter is tested against all supported versions of Python and `Django`__,
-as well as the latest versions of Django REST Framework (`DRF`__).
+as well as the latest version of Django REST Framework (`DRF`__).
 
 __ https://www.djangoproject.com/download/
 __ http://www.django-rest-framework.org/
@@ -24,4 +24,4 @@ __ http://www.django-rest-framework.org/
 
 * **Python**: 2.7, 3.3, 3.4, 3.5
 * **Django**: 1.8, 1.9, 1.10
-* **DRF**: 3.4, 3.5
+* **DRF**:  3.5

--- a/tox.ini
+++ b/tox.ini
@@ -1,8 +1,7 @@
 [tox]
 envlist =
-       {py27,py33,py34,py35}-django18-restframework{34,35},
-       {py27,py34,py35}-django{19,110}-restframework{34,35},
-       {py27,py34,py35}-django111-restframework35,
+       {py27,py33,py34,py35}-django18-restframework35,
+       {py27,py34,py35}-django{19,110,111}-restframework35,
        {py35}-djangolatest-restframeworklatest,
        warnings
 
@@ -18,7 +17,6 @@ deps =
     django110: django>=1.10.0,<1.11.0
     django111: django>=1.11a1,<2.0
     djangolatest: https://github.com/django/django/archive/master.tar.gz
-    restframework34: djangorestframework>=3.4,<3.5
     restframework35: djangorestframework>=3.5,<3.6
     restframeworklatest: https://github.com/tomchristie/django-rest-framework/archive/master.tar.gz
     -rrequirements/test-ci.txt

--- a/tox.ini
+++ b/tox.ini
@@ -2,8 +2,8 @@
 envlist =
        {py27,py33,py34,py35}-django18-restframework{34,35},
        {py27,py34,py35}-django{19,110}-restframework{34,35},
-       {py27,py34,py35,py36}-django111-restframework35,
-       {py35,py36}-djangolatest-restframeworklatest,
+       {py27,py34,py35}-django111-restframework35,
+       {py35}-djangolatest-restframeworklatest,
        warnings
 
 [testenv]

--- a/tox.ini
+++ b/tox.ini
@@ -8,7 +8,7 @@ envlist =
 [testenv]
 commands = coverage run --parallel-mode --source django_filters ./runtests.py {posargs}
 ignore_outcome =
-    django{111,latest}: True
+    djangolatest: True
 setenv =
        PYTHONDONTWRITEBYTECODE=1
 deps =

--- a/tox.ini
+++ b/tox.ini
@@ -1,7 +1,8 @@
 [tox]
 envlist =
        {py27,py33,py34,py35}-django18-restframework{34,35},
-       {py27,py34,py35}-django{19,110,111}-restframework{34,35},
+       {py27,py34,py35}-django{19,110}-restframework{34,35},
+       {py27,py34,py35}-django111-restframework35,
        {py35}-djangolatest-restframeworklatest,
        warnings
 

